draft-ietf-openpgp-pqc.md

```diff
@@ -1632,61 +1571,52 @@ Implementations may also make more nuanced decisions.
 
 # Additional considerations
 
-## Performance Considerations for SLH-DSA {#performance-considerations}
+## Performance Considerations for SLH-DSA-SHAKE {#performance-considerations}
 
-This specification introduces both ML-DSA + ECC as well as SLH-DSA as PQ(/T)
+This specification introduces both ML-DSA + ECC as well as SLH-DSA-SHAKE as PQ(/T)
 signature schemes.
 
 Generally, it can be said that ML-DSA + ECC provides a performance in terms of
 execution time requirements that is close to that of traditional ECC signature
 schemes.
 Regarding the size of signatures and public keys, though, ML-DSA has far greater
 requirements than traditional schemes like EC-based or even RSA
-signature schemes. Implementers may want to offer SLH-DSA for applications
-where a higher degree of trust in the signature scheme is required. However,
-SLH-DSA has performance characteristics in terms of execution time of the
-signature generation as well as space requirements for the signature that are
-even greater than those of
-ML-DSA + ECC signature schemes.
-
-Pertaining to the execution time, the particularly costly operation in SLH-DSA
-is the signature generation. In order to achieve short signature generation
-times, one of the parameter sets with the name ending in the letter "f" for
-"fast" should be chosen. This comes at the expense of a larger signature size.
-
-In order to minimize the space requirements of a SLH-DSA signature, a parameter
-set ending in "s" for "small" should be chosen. This comes at the expense of a
-longer signature generation time.
-
-# IANA Considerations
-
-IANA is requested to add the following registries to the `OpenPGP`
-registry group at https://www.iana.org/assignments/openpgp:
-
-- Registry name: `OpenPGP SLH-DSA-SHA2 parameters`
-
-  Registration procedure: SPECIFICATION REQUIRED [RFC8126]
-
-  The registry contains the values defined in {{slhdsa-param-sha2}} in this document.
-
-- Registry name: `OpenPGP SLH-DSA-SHAKE parameters`
+signature schemes.
 
-  Registration procedure: SPECIFICATION REQUIRED [RFC8126]
+Implementers may want to offer SLH-DSA-SHAKE for applications where the milder
```

Comment:
It's not clear to me exactly what "milder" security assumptions are supposed to mean.  For example, If it means "less stringent threat model" , then that suggests that these signatures are less robust.  But if it means "less scary threat model", then that suggests that these signatures are more robust.

Reply:
Right, "milder" is not clear enough. I replaced it by "weaker". If it should not be clear how to understand this term, it is explained [here](https://crypto.stackexchange.com/questions/60059/comparing-stronger-vs-weaker-assumptions).